### PR TITLE
Update whats new

### PR DESCRIPTION
**Pull request by jcalcaben**
## Purpose of this pull request

This pull request (PR) updates whats new since July 13

### CODE REVIEW

File: src/_data/whats-new.yml [hunk 1]
```diff
@@ -4,8 +4,106 @@ description:
...
+  type: Technical
+  date: July 17, 2020
+  link: https://github.com/magento/devdocs/pull/7529
+- description: Updated _Cloud Guide_ with information about account and project security
+    options:<br/>- Added instructions to configure 2FA security and API tokens setting
+    to the [Manage user access](https://devdocs.magento.com/cloud/project/user-admin.html)
+    topic.<br/>- Added [Enable MFA enforcement for SSH](https://devdocs.magento.com/cloud/project/project-enable-mfa-project.html)
+    topic to describe this project-level feature to enforce 2FA requirement for SSH
+    access to Magento Cloud environments. Also added instructions for authenticating
+    on projects that have MFA enforcement enabled.<br/>
```
Comment (hguthrie) [inline src/_data/whats-new.yml:97]:
```suggestion
    on projects that have MFA enforcement enabled.
```

File: src/_data/whats-new.yml [hunk 2]
```diff
@@ -4,8 +4,106 @@ description:
...
+  type: New Topic
+  date: July 23, 2020
+  link: https://github.com/magento/devdocs/pull/7566
+- description: Updated the [Magento Cloud CLI command reference for v1.34.4](https://devdocs.magento.com/guides/v2.4/reference/cli/magento-cloud.html).
+  versions: 2.x
+  type: Technical
+  date: July 22, 2020
+  link: https://github.com/magento/devdocs/pull/7572
+- description: Because two-factor authentication for admin accounts is required in
+    2.4.0, references to the `POST /V1/integration/admin/token`endpoint have been
```
Comment (hguthrie) [inline src/_data/whats-new.yml:49]:
```suggestion
    2.4.0, references to the `POST /V1/integration/admin/token` endpoint have been
```

File: src/_data/whats-new.yml [hunk 3]
```diff
@@ -4,8 +4,106 @@ description:
...
+    replaced  with `POST /V1/tfa/provider/google/authenticate` in the [REST tutorials](https://devdocs.magento.com/guides/v2.4/rest/tutorials/prerequisite-tasks/create-admin-token.html)
```
Comment (hguthrie) [inline src/_data/whats-new.yml:50]:
```suggestion
    replaced with `POST /V1/tfa/provider/google/authenticate` in the [REST tutorials](https://devdocs.magento.com/guides/v2.4/rest/tutorials/prerequisite-tasks/create-admin-token.html)
```

File: src/_data/whats-new.yml [hunk 4]
```diff
@@ -4,8 +4,106 @@ description:
...
+  type: Technical
+  date: July 24, 2020
+  link: https://github.com/magento/devdocs/pull/7584
+- description: Added additional troubleshooting instructions for [PrivateLink VPC
+    connections](https://devdocs.magento.com/cloud/project/privatelink-service.html).
+  versions: 2.x
+  type: Technical
+  date: July 23, 2020
+  link: https://github.com/magento/devdocs/pull/7538
+- description: Adds support for security.txt to help security researchers report potential
```
Comment (hguthrie) [inline src/_data/whats-new.yml:26]:
```suggestion
- description: Adds support for `security.txt` to help security researchers report potential
```

**Comment by jcalcaben** [comment]
running tests

**Comment by jcalcaben** [comment]
running tests

**Pull request merged**